FEXCore/Source/Interface/IR/Passes/x87StackOptimizationPass.cpp

```diff
@@ -1044,7 +1044,7 @@ void X87StackOptimization::Run(IREmitter* Emit) {
 
       case OP_INCSTACKTOP: {
         if (SlowPath) {
-          UpdateTopForPush_Slow();
+          UpdateTopForPop_Slow();
```

Comment:
Good catch... 
Not only I can't believe I missed this, but also that it was not caught by other tests.